core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/DefaultConfigs.scala

```diff
@@ -0,0 +1,16 @@
+package org.virtuslab.ideprobe.dependencies
+
+object DefaultConfigs {
+  val defaultIntellijConfig: String =
+    s"""
+       |probe {
```

Comment:
Nit: can it be just read from resources (using some lower-level API like [`Source.fromResource`](https://www.baeldung.com/scala/read-file-from-resources#using-source), not `ConfigFactory` or pureconfig)?